### ISSUE TITLE
Data Transfer via Export - Append to Existing File

### ISSUE THREAD

**Issue by SPalmerLS** [dbeaver/dbeaver#13934]
There is no problem, but currently DBeaver will export to a file (in my case .csv), but if I wish to export to an already existing .csv file, the file gets overwritten. What I'd like to see is the option to append to a file on export, OR for DBeaver to realize that the file already exists on export and adds the export data to the first sheet, OR DBeaver will give you the option to append to an existing file but (for excel) appends to a new sheet in an already existing workbook.

Currently I have to export daily to separate workbooks and manually append the data.

**Comment by tati-kru** [comment]
We're going to implement that for sure. Thank you for the idea.

**Comment by uslss** [comment]
1. Checkbox cannot be unchecked
![k79pi06Pxb](https://user-images.githubusercontent.com/31996417/146146563-61190ca2-2adb-4c8b-9f25-b39f39cac797.gif)

2. Checkbox isn't greyed out in case Copy to clipboard option is enabled.
![nS0jgMo1rw](https://user-images.githubusercontent.com/31996417/146146693-d2f272dd-398b-4312-a7b9-646240a0aef6.gif)

**Comment by uslss** [comment]
incorrect behavior is shown In the case of multi-table export to the single file with append option enabled.
**STR**
1. Start export of the three tables (t1, t2, t3) and perform it to the single file (we've got file with t1, t2. t3 tables one-by-one)
2. Add a new row to the first table (t1) and perform an export to the same file with the append option enabled
**Actual result**
We've got a file with the structure like t1,t2,t3 than t1 with added row, t2, t3.
